plottable.d.ts

```diff
@@ -204,17 +208,14 @@ declare module Plottable {
              */
             has(key: K): boolean;
             /**
-             * Return an array of the values in the Map
-             *
-             * @return {V[]} The values in the store
-             */
-            values(): V[];
-            /**
-             * Return an array of keys in the Map.
+             * The forEach method executes the provided callback once for each key of the map which
+             * actually exist. It is not invoked for keys which have been deleted.
+             * However, it is executed for values which are present but have the value undefined.
```

Comment:
Can we simplify the doc here to be
"Shim for ES6 Map.forEach" and then put the line to `forEach` on a new line